.env.sample

```diff
@@ -11,6 +11,7 @@ PRIVATE_KEYS="0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
 NO_SETUP=false
 USE_SHIVA=false
 NETWORK_CONFIG=./networkContext.json
+TEST_TIMEOUT=30000
```

Comment:
Can we make this longer than the timeout time on the nodes just to give them a chance to actually time out server-side before we give up?  How about 45 seconds to give a little bit of time for setup costs plus 30s for the nodes to time out?  If the reason why it's taking >30s for the test to finish, it would be better if we got an error _from the nodes timing out on the lit action_ than a generic 'local' test timeout from our test suite.

Reply:
Yes sir! 🫡